examples/blog-basic/mkdocs.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2016-2023 the authors:
+#
+#   Martin Donath <martin.donath@squidfunk.com>
+#   Alex Voss <alex@corealisation.com>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+# -----------------------------------------------------------------------------
+# Recommended: set up configuration validation, see https://t.ly/xpZXU
+# -----------------------------------------------------------------------------
+
+# Project information
+site_name: Basic blog setup example
+
+# Theme
+theme:
+  name: material
+
+# Plugins 
+# Note that you need to add the search plugin manually
+# as soon as you use another plugin.
+plugins:
+  - search
+  - blog:
+      pagination: true
```

Comment:
Both settings can be removed, as we only have 2 blog posts (with a default of 10 per page) and pagination is enabled by default. I'd say we keep it as minimal as possible.